services.yaml

```diff
@@ -1610,6 +1610,49 @@ paths:
           $ref: "#/responses/UnauthorizedError"
         404:
           $ref: "#/responses/NotFoundError"
+  /services/{env_id}/settings/alerts/service_now:
+    get:
+      tags:
+        - Alerts
+      summary: "Fetch ServiceNow Tables"
+      description: "Use this call to Load ServiceNow tables associated with specific credentials"
+      parameters:
+        - in: path
+          name: env_id
+          description: Environment to use
+          type: string
+          required: true
+        - in: body
+          name: credentials
+          description: ServiceNow Credentials
+          schema:
+            type: object
+            properties:
+              url:
+                type: "string"
+                example: "http://servicenow.company.com/hooks"
+              user_id:
+                type: "string"
+                example: "admin"
+              password:
+                type: "string"
+                example: "pa$$1"
+      responses:
+        200:
+          description: List of ServiceNow tables
+          schema:
+            type: object
+            properties:
+              tables:
+                type: array
```

Comment:
The response is an array of an object of a `ServiceNow Table` which contains the table's display name (dubbed as label) and the table's internal name (dubbed as name and is also the table's ID)